core/iwasm/aot/aot_loader.c

```diff
@@ -262,47 +262,35 @@ load_string(uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
     uint16 str_len;
 
     CHECK_BUF(p, p_end, 1);
```

Comment:
Can remove L264, as p is checked in read_uint16

Reply:
done